docs/conf.py

```diff
@@ -155,7 +154,7 @@ def wrapper(decorator):
 # html_theme_options = {}
 
 # Add any paths that contain custom themes here, relative to this directory.
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+# html_theme_path = []
```

Comment:
It's because of this warning from Sphinx. We are treating warnings as errors.
```
WARNING: Calling get_html_theme_path is deprecated. If you are calling it to define html_theme_path, you are safe to remove that code.
```
But I think we can leave the dependency because we still have
``html_theme = "sphinx_rtd_theme"``

I think back in the day the theme loading needed extra help that isn't required anymore. But building locally, with the changes, the docs look the same as on Read the Docs.

Reply:
ok, great! Thanks for figuring that out!